extractor/src/main/java/org/schabi/newpipe/extractor/services/bandcamp/extractors/BandcampSearchExtractor.java

```diff
@@ -50,40 +50,31 @@ public boolean isCorrectedSearch() {
 
     public InfoItemsPage<InfoItem> getPage(final Page page)
             throws IOException, ExtractionException {
-        final String html = getDownloader().get(page.getUrl()).responseBody();
-
         final MultiInfoItemsCollector collector = new MultiInfoItemsCollector(getServiceId());
+        final Document d = Jsoup.parse(getDownloader().get(page.getUrl()).responseBody());
 
-
-        final Document d = Jsoup.parse(html);
-
-        final Elements searchResultsElements = d.getElementsByClass("searchresult");
-
-        for (final Element searchResult : searchResultsElements) {
-
-            final String type = searchResult.getElementsByClass("result-info").first()
-                    .getElementsByClass("itemtype").first().text();
+        for (final Element searchResult : d.getElementsByClass("searchresult")) {
+            final String type = searchResult.getElementsByClass("result-info").stream()
+                    .flatMap(element -> element.getElementsByClass("itemtype").stream())
+                    .map(Element::text)
+                    .findFirst()
+                    .orElse("");
 
             switch (type) {
                 default:
-                    continue;
```

Comment:
1. shouldn't be the ``default`` at the end? (Code-Style)
2. this will cause the ``default`` to behave in the same ways as  ``case "FAN"`` which is wrong

Reply:
I just want `default:` to do the same thing as `"FAN"`, that is, exiting from the `switch` (`continue;` and `break;` result in the same thing happening here). But yes, I will move the default at the end.